include/xtensor/xaccumulator.hpp

```diff
@@ -198,11 +198,10 @@ namespace xt
         inline auto accumulator_impl(F&& f, E&& e, std::size_t axis, evaluation_strategy::immediate_type)
         {
             using init_type = typename F::init_value_type;
-            using init_functor_type = typename F::init_functor_type;
```

Comment:
What is the issue with this type definition? Does it generate an warning because it is never used? Or does it simply not exist?

Reply:
It's never used. Clang would complain when -Wunused-local-typedef and -Werror is used.

Reply:
Oh, I read too fast, I think it was comlaining about the typedef in the class definition (that would have been a false positive since it's part of the public API that might be used by downstream projects).